README.md

```diff
@@ -40,6 +40,13 @@ The addons-linter will check your add-on and show you errors, warnings, and frie
 addons-linter --help
 ```
 
+#### Privileged extensions
+
+The addons-linter can lint **privileged extensions** using the `--privileged` CLI flag. This option changes the behavior of the linter to:
```

Comment:
Not a big dea, but I wonder if this should be actually stating the opposite (something like: "For linting privileged extensions addons-linter should be executed by including the `--privileged` CLI flag").